Formula/liblwgeom.rb

```diff
@@ -27,14 +14,15 @@ class Liblwgeom < Formula
 
   head do
     url "https://svn.osgeo.org/postgis/trunk/"
-    depends_on "postgresql" => :build # don't maintain patches for HEAD
+    depends_on "postgresql" => :build
```

Comment:
Do we still need this now that `-with-pgconfig` was removed?
Other than that, it looks good to me!

Reply:
Definitely not needed any more, just removed it.